gwpy/frequencyseries/tests/test_frequencyseries.py

```diff
@@ -168,14 +168,16 @@ def test_interpolate(self):
         # create a simple FrequencySeries
         df, nyquist = 1, 256
         nsamp = int(nyquist/df) + 1
-        fseries = FrequencySeries(numpy.ones(nsamp), f0=0, df=df, unit='')
+        fseries = FrequencySeries(numpy.ones(nsamp), f0=1, df=df, unit='')
 
         # create an interpolated FrequencySeries
         newf = fseries.interpolate(df/2.)
 
         # check that the interpolated series is what was expected
         assert newf.unit == fseries.unit
         assert newf.size == 2*(fseries.size - 1) + 1
+        assert newf.df.value == df / 2.
+        assert newf.f0.value == fseries.f0.value
```

Comment:
Can we just compare the quantities, to make sure the units are mangled as well:

```python
        assert new.df == fseries.df / 2.
        assert new.f0 == fseries.f0
```
?